src/test/ReactTestUtils.js

```diff
@@ -186,9 +184,14 @@ var ReactTestUtils = {
     }
     return ReactTestUtils.findAllInRenderedTree(root, function(inst) {
       if (ReactTestUtils.isDOMComponent(inst)) {
-        var classList = ReactDOM.findDOMNode(inst).className.split(/\s+/);
-        return classNames.every(function(className) {
-          return classList.indexOf(className) !== -1;
+        var className = inst.className;
+        if (!className && typeof className !== 'string') {
```

Comment:
Maybe I'm missing something, but this line looks suspicious to me.  Is the boolean of a string ever falsey?